### DISCUSSION TITLE
Welcome to WebGAL Discussions!

### DISCUSSION THREAD

**Discussion by MakinoharaShoko** [OpenWebGAL/WebGAL#48] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 2439905184** [comment]
好耶

**Comment by twiso100** [comment]
想問一下  我大概做了一個gal遊戲  但電腦 android手機都可以使用 不過ios系統 完全不行...這種要怎麼修改?

我不確定我是不是漏掉了什麼資訊  想請問一下開發者  感謝

**Reply by MakinoharaShoko** [reply]
iOS 现在没有经过调试，所以可能存在一些兼容性问题。可能需要等待使用iOS的开发者协助维护，才能解决这些问题。

**Reply by MakinoharaShoko** [reply]
> iOS 现在没有经过调试，所以可能存在一些兼容性问题。可能需要等待使用iOS的开发者协助维护，才能解决这些问题。

嗯，因为我没有 iOS 设备，就是这样（如果我个人使用iPhone的话，我想一定会重视这些问题的，但现在的问题是我不是很方便调试）。

**Reply by twiso100** [reply]
我自己也沒iPhone 但朋友很多都用...也不方便借到  不然就可以慢慢測..

https://www.youtube.com/shorts/oPU2Vbbhul0
目前的狀況大概是這樣 按了幾次不行後就壞掉 不過同時間安卓和電腦都是沒問題

[铃色☆记忆] http://hoshinasuzu.cn/   他可能有改了些html碼  我朋友iPhone手機倒是可以執行
雖然畫面匹配也是有點問題 

感覺就是index.html 可能有出現什麼bug 不確定  我直接丟給chatgpt 他也說沒問題 但有幫忙優化

總之我在想看看有沒有平衡解  不過我讀生物的 代碼不太熟 我單純就反映狀況  你看看就好 .

**Reply by MakinoharaShoko** [reply]
收到你的反馈了，如果有机会的话，可能我去租个 iPhone 测试一下
